Error: Package path . is not exported from package htmlparser2

Hi,

I'm getting this while compiling with webpack5:
```
ERROR in ./node_modules/@humanspeak/svelte-markdown/dist/utils/token-cleanup.js 1:0-37
Module not found: Error: Package path . is not exported from package /node_modules/@humanspeak/svelte-markdown/node_modules/htmlparser2 (see exports field in /node_modules/@humanspeak/svelte-markdown/node_modules/htmlparser2/package.json)
 @ ./node_modules/@humanspeak/svelte-markdown/dist/SvelteMarkdown.svelte 28:0-60 45:22-38
 @ ./node_modules/@humanspeak/svelte-markdown/dist/index.js 1:0-53 2:15-29
```

--- a/.coderabbit.yaml
+++ b/.coderabbit.yaml
@@ -1,0 +1,140 @@
+language: en-US
+tone_instructions: ''
+early_access: false
+enable_free_tier: true
+reviews:
+    profile: assertive
+    request_changes_workflow: false
+    high_level_summary: true
+    high_level_summary_placeholder: '@coderabbitai summary'
+    high_level_summary_in_walkthrough: false
+    auto_title_placeholder: '@coderabbitai'
+    auto_title_instructions: ''
+    review_status: true
+    commit_status: true
+    fail_commit_status: false
+    collapse_walkthrough: false
+    changed_files_summary: true
+    sequence_diagrams: true
+    assess_linked_issues: true
+    related_issues: true
+    related_prs: true
+    suggested_labels: true
+    auto_apply_labels: false
+    suggested_reviewers: true
+    auto_assign_reviewers: false
+    poem: true
+    labeling_instructions: []
+    path_filters: []
+    path_instructions: []
+    abort_on_close: true
+    disable_cache: false
+    auto_review:
+        enabled: true
+        auto_incremental_review: true
+        ignore_title_keywords: []
+        labels: []
+        drafts: false
+        base_branches: []
+    finishing_touches:
+        docstrings:
+            enabled: true
+        unit_tests:
+            enabled: true
+    tools:
+        ast-grep:
+            rule_dirs: []
+            util_dirs: []
+            essential_rules: true
+            packages: []
+        shellcheck:
+            enabled: true
+        ruff:
+            enabled: true
+        markdownlint:
+            enabled: true
+        github-checks:
+            enabled: true
+            timeout_ms: 90000
+        languagetool:
+            enabled: true
+            enabled_rules: []
+            disabled_rules: []
+            enabled_categories: []
+            disabled_categories: []
+            enabled_only: false
+            level: default
+        biome:
+            enabled: true
+        hadolint:
+            enabled: true
+        swiftlint:
+            enabled: true
+        phpstan:
+            enabled: true
+            level: default
+        golangci-lint:
+            enabled: true
+        yamllint:
+            enabled: true
+        gitleaks:
+            enabled: true
+        checkov:
+            enabled: true
+        detekt:
+            enabled: true
+        eslint:
+            enabled: true
+        rubocop:
+            enabled: true
+        buf:
+            enabled: true
+        regal:
+            enabled: true
+        actionlint:
+            enabled: true
+        pmd:
+            enabled: true
+        cppcheck:
+            enabled: true
+        semgrep:
+            enabled: true
+        circleci:
+            enabled: true
+        sqlfluff:
+            enabled: true
+        prismaLint:
+            enabled: true
+        oxc:
+            enabled: true
+        shopifyThemeCheck:
+            enabled: true
+chat:
+    auto_reply: true
+    integrations:
+        jira:
+            usage: auto
+        linear:
+            usage: auto
+knowledge_base:
+    opt_out: false
+    web_search:
+        enabled: true
+    learnings:
+        scope: auto
+    issues:
+        scope: auto
+    jira:
+        usage: auto
+        project_keys: []
+    linear:
+        usage: auto
+        team_keys: []
+    pull_requests:
+        scope: auto
+code_generation:
+    docstrings:
+        language: en-US
+        path_instructions: []
+    unit_tests:
+        path_instructions: []

--- a/src/lib/utils/token-cleanup.ts
+++ b/src/lib/utils/token-cleanup.ts
@@ -1,4 +1,4 @@
-import { Parser } from 'htmlparser2'
+import * as htmlparser2 from 'htmlparser2'
 import type { Token } from 'marked'
 
 /**
@@ -128,7 +128,7 @@ export const parseHtmlBlock = (html: string): Token[] => {
         /^(br|hr|img|input|link|meta|area|base|col|embed|keygen|param|source|track|wbr)$/i
     const openTags: string[] = []
 
-    const parser = new Parser(
+    const parser = new htmlparser2.Parser(
         {
             onopentag: (name, attributes) => {
                 if (currentText.trim()) {

